app/src/main/res/layout/fragment_login.xml

```diff
@@ -116,54 +127,79 @@
                     app:onEditorActionClicked="@{()->viewModel.onClick()}" />
             </com.google.android.material.textfield.TextInputLayout>
 
-            <TextView
-                android:id="@+id/textView3"
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/tvForgetPass"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginStart="8dp"
-                android:layout_marginEnd="8dp"
-                android:text="@string/forgot_your_password"
-                android:textColor="@color/black"
-                android:textSize="14sp"
+                android:layout_marginTop="@dimen/margin_10"
+                android:text="@string/forgot_password"
+                android:textColor="@color/grey"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/field_password" />
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/btnLogin"
+                android:layout_width="match_parent"
```

Comment:
layout_width define "0dp" and layout_constraintStart_toStartOf="parent"  app:layout_constraintEnd_toEndOf="parent"